auth/gcloud/aio/auth/token.py

```diff
@@ -132,12 +143,20 @@ def get_service_data(
         return {}
 
 
-class Token:
+@dataclass
+class TokenResponse:
+    value: str
+    expires_in: int = GCLOUD_TOKEN_DURATION
```

Comment:
Looks like there are fairly limited places where this default gets used, all within the new IAP flow. It might be a bit easier to reason about if this value is passed in explicitly in that case, since (if I'm reading this correctly!) it looks like there might be two values at play here: previously, we passed this value in and Google created tokens of the designated duration. With IAP, though, it seems like we're finding the tokens have this duration even though we aren't passing this value in (for gce metadata and authorized users)?

Reply:
For some reason the responses from IAP token requests are all over the place and they don't provide you with an expiry. Maybe there's some way to force them to, but in practice the tokens do expire within 1 hour.

I've moved this default TTL to the class-level and removed the default from this `TokenResponse` dataclass. LMK what you think!